src/provider/model/AppDirectory.ts

```diff
@@ -1,25 +1,45 @@
-import {injectable} from 'inversify';
+import {injectable, inject} from 'inversify';
+import {MaskWatch} from 'openfin-service-config/Watch';
+import {Scope} from 'openfin-service-config/Types';
+import {ScopedConfig} from 'openfin-service-config';
 
+import {Inject} from '../common/Injectables';
 import {Application, AppName} from '../../client/directory';
 import {AppIntent} from '../../client/main';
+import {ConfigurationObject} from '../../../gen/provider/config/fdc3-config';
+
+import {ConfigStore} from './ConfigStore';
 
 enum StorageKeys {
     URL = 'fdc3@url',
     APPLICATIONS = 'fdc3@applications'
 }
 
 // Demo development app directory url
-const devUrl = 'http://localhost:3923/provider/sample-app-directory.json';
-
+export const devAppDirectoryUrl = 'http://localhost:3923/provider/sample-app-directory.json';
```

Comment:
Should we make this all caps to match other `export const`s?

Reply:
Yes, I like that much better.

Reply:
Fixed